mysite/mysite/urls.py

```diff
@@ -15,10 +15,14 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path, include
-
+from django.urls import include, path
+from polls.views import QuestionAPIView, QuestionCreateAPIView
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('hello/', include('polls.urls'))
+    path("admin/", admin.site.urls),
+    path("hello/", include("polls.urls")),
+    path("api/questions/", QuestionAPIView.as_view(), name="question-list"),
+    path(
+        "api/questions/create/", QuestionCreateAPIView.as_view(), name="question-create"
```

Comment:
path("api/", include("polls.urls")),
**move the remaining two apis to polls application**